docs/pages/accounts/sso.mdx

```diff
@@ -1,62 +1,72 @@
+> The Expo SSO system currently formally supports [Okta](https://www.okta.com/), but we implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, let us know.
 
 ## SSO user sign in
 
 ### Expo website
 
-1. Navigate to https://expo.dev/sso-login and enter the account name of your organization.
+1. Navigate to [expo.dev/sso-login](https://expo.dev/sso-login) and enter the account name of your organization.
 
-    > You can create a link that pre-fills the organization name. For example, https://expo.dev/sso-login/test-org will pre-fill `test-org`.
+    > You can create a link that pre-fills the organization name. For example, [expo.dev/sso-login/test-org](https://expo.dev/sso-login/test-org) pre-fills `test-org`.
 
 1. Log in to your identity provider (IdP).
-1. You will then be asked to choose a Expo username. This will be your username in Expo.
+1. Next, you'll be prompted to select an Expo username. This will be the username for your Expo account.
 
 ### Expo CLI
 
-`npx expo login --sso`
+When using the Expo CLI, you can run the following command to log in to your Expo account.
+
+<Terminal cmd={["$ npx expo login --sso"]} />
+
+You will be prompted to log in via the Expo website in a browser and will be redirected back to the CLI upon completion.
 
 ### EAS CLI
 
-`eas login --sso`
+When using the EAS CLI, you can run the following command to log in to your Expo account.
+
+<Terminal cmd={["$ eas login --sso"]} />
+
+You will be prompted to log in via the Expo website in a browser and will be redirected back to the CLI upon completion.
 
 ### Expo Go
 
-1. Go through normal sign in flow but click "Continue with SSO" button on sign in page.
-1. Follow steps for Expo website above.
+1. Click the **Continue with SSO** button on the sign-in page when going through the sign-in flow.
+1. Follow the above steps for signing in to the Expo website.
 
 ## SSO user restrictions
 
-SSO users are just like regular users for the most part, but there are a few known exceptions:
+SSO users are like regular users. However, there are a few known exceptions:
 - SSO users can only belong to their SSO organization. They also cannot create additional organizations.
 - SSO users cannot leave their SSO organization. Doing so deletes their SSO user.
-- SSO users cannot access the Expo forums.
+- SSO users cannot log in to the Expo forums.
 - SSO users cannot subscribe to EAS for their personal accounts.
 
 ## SSO administration
 
-### Removing SSO users
+Both new organizations and existing organizations are supported. Organizations with existing non-SSO users can enable SSO and both non-SSO users and SSO users can continue to be added. Migration of non-SSO users to SSO users is not currently supported.
```

Comment:
I could see it the other way, but if it's the case that an SSO user can be added with the same email as a non-SSO user, it'd be my preference to document it. I feel like this is going to be the way existing orgs evaluate and/ or migrate to SSO initially. I can make it part of an internal script, but if it's part of the documentation, less need for someone to ask how they should migrate themselves and less room for me to second-guess myself here.

That's assuming this is what we would encourage them to do at all and don't feel like there's too much risk of confusion. I think it's worth a small-ish existing org to consider, because otherwise a key user can only get feedback about SSO by asking new org members how it's going (way slower feedback loop, less confidence in the offering).

Reply:
I don't think we should encourage migration necessarily. Maybe it's a terminology thing. I guess I would describe it as "enabling SSO as a sign in option to the org" versus "migrating the org to SSO". So the person evaluating it would just be creating a new expo SSO user, but that's not what we should recommend for all their users to do.

By design, it is fine to have both SSO users and non-SSO users, and yes, they can have the same email. As for evaluation, I could see it as happening in the following order:
1. We set it up for them
2. Someone on their IT team tries it out with their account on their IdP (it can have the same email as their existing expo user account)
3. They decide to leave SSO enabled, and only new members are added via SSO. Old members can keep working as normal.

Reply:
I'll document this a bit.

Reply:
One other important thing is that SSO users don't know they have an email. It's an implementation detail, and not all SSO users do have an email either.

Reply:
In the sort term I'm more focused on existing key users to be able to use SSO themselves, not necessarily migration. If I tell key user at partner X that SSO is ready for them, but it's just for new accounts, their first question is going to be, "well, how do I try it out?" Noting the second account path is possible covers that and at least hints at an informal migration strategy that could be pursued before we offer something more formal.

Even if it's not technically required, I think full migration/ adoption will be the goal for many orgs. Without full migration, these orgs do not get the full benefits of SSO (ability to revoke access, manage password/ MFA policies across, etc. for all users from their central identity provider).

Reply:
Yeah, that makes sense. I think we'll iron this out through the first couple onboardings and go from there. The preview release will definitely not be super smooth.